mw-webapp/src/dataAccessLogic/getUsersPreview.ts

```diff
@@ -15,8 +14,12 @@ export const getUsersPreview = async (): Promise<UserPreview[]> => {
   const firstUser = usersDTO[0];
 
```

Comment:
Method shouldn't work only with first user. I think need create method that will get user by uuid and second methos for get all users. But not just first user)

Reply:
Done